subsys/bluetooth/controller/ll_sw/ll_master.c

```diff
@@ -22,11 +22,15 @@ u32_t ll_create_connection(u16_t scan_interval, u16_t scan_window,
 {
 	u32_t status;
 
+	if (radio_scan_is_enabled()) {
+		return BT_HCI_ERR_CMD_DISALLOWED;
+	}
+
 	status = radio_connect_enable(peer_addr_type, peer_addr, interval,
 				      latency, timeout);
 
 	if (status) {
-		return status;
+		return BT_HCI_ERR_CMD_DISALLOWED;
 	}
 
 	return radio_scan_enable(0, own_addr_type,
```

Comment:
No, it can still return error if it fails to acquire the initiator memory. I should probably return a better value in that case